### ISSUE TITLE
BUG Ore A# discrepancy

### ISSUE THREAD

**Issue by DragooKnight** [doomeer/factorio#39]
When trying to find out how much ore a specific number of miners will make with A#, the number of miners and resulting ore appears to be less-than-halved. 
Compare the A# output of iron plate and steel in _example one_ which uses less electric miners than the number in **example two's** iron ore field (from one of my new iron patches), which itself outputs less miners and ore production than _example one_ uses.
(how coherent was my report?  I'm new to this)
![what is with](https://user-images.githubusercontent.com/22573589/29384241-5da290d0-82a1-11e7-9b5a-70fce9f7e7b4.png)
![these miner values](https://user-images.githubusercontent.com/22573589/29384350-c1aa8588-82a1-11e7-9144-bd1b91ef837c.png)

**Comment by firefly762** [comment]
Just erased a very long explanation because I just realized what you're trying to relay. So you're trying to find out how many items/sec 163 electric drills will output on an iron ore patch, correct? And inputting it into the calculator it seems to divide your 163 input by just over two for some reason (163/2.0999742334). Is that an accurate abbreviation of your issue?

If I'm correct (forgive me if I'm not and my breath is about to be wasted), I'm not sure why the calculator does that, but you are correct something doesn't seem right. As for how much your drills will put out, the actual formula factorio uses is (MP-MH)*MS/MT = items/sec/drill.
MP = Mining Power
MT =       "      Time
MS =       "      Speed
MH =      "      Hardness
For iron ore it would come out as (3-0.9)*0.5/2 = 0.525 iron ore/sec*163 drills = 85.575 iron/sec

Hope that helps...and makes sense :/

Cheers

**Comment by DragooKnight** [comment]
Yes, it seems to be multiplying the number of Specified Total Drills by the PPS, then getting the PPS of that number.  So basically it runs the algorithm once internally, then recursively once for the output?
[Because, of course, when you multiply a number by a 0.decimal, you get LESS than what you put in]
[(MP-MH)*MS/MT=PPS]
